kube/manifests/prometheus-gcs.yaml

```diff
@@ -52,7 +52,7 @@ spec:
           mountPath: /var/prometheus
       - name: thanos-sidecar
         # Always use explicit image tags (release or master-<date>-sha) instead of ambigous `latest` or `master`.
-        image: improbable/thanos:v0.1.0
+        image: improbable/thanos:v0.2.1
```

Comment:
Same here for --objstore.config=<bucket.config-yaml> 

We might want to change gossip to DNS, but we can do it later.